src/app/modules/users/components/users-list/users-list.component.spec.ts

```diff
@@ -14,7 +14,17 @@ describe('UsersListComponent', () => {
   const actionSub: ActionsSubject = new ActionsSubject();
 
   const state: UserState = {
-    data: [{ name: 'name', email: 'email', role: 'role', id: 'id', tenant_id: 'tenant id', deleted: 'delete' }],
+    data: [
+      {
+        name: 'name',
+        email: 'email',
+        role: null,
+        roles: ['admin', 'test'],
```

Comment:
That's temporary. We will remove `role` when the whole feature around `roles` is completed. Right now is under a FF in the backend, but the frontend would accept both for now without any issues in case we switch to use one or the other field.

Reply:
Got it. It would be great if you add a TODO and link to the issue which is going to remove the role attribute.